presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -755,6 +755,42 @@ Procedures
     Unregisters given, existing partition in the metastore for the specified table.
     The partition data is not deleted.
 
+Special Columns
+---------------
+
+In addition to the defined columns, the Hive connector automatically exposes
+metadata in a number of hidden columns in each table. You can use these columns
+in your SQL statements like any other column, e.g. to return the data or to use
+them in condition statements.
+
+* ``$bucket``
+    Name of the bucket.
+
+* ``file_modified_time``
+    Date and time of the last modification of the file in the storage.
+
+* ``file_size``
+    Size of the file in the storage.
+
+* ``$path``
+    Full path to the file in the storage.
+
+* ``$shard_row_id``
```

Comment:
fyi @electrum  .. not sure if this should be documented and if it is correct as it stands.

Reply:
I just looked at the source a bit..

Reply:
I believe `shard_row_id` is an synthetic column used during delete operations, and not a real column you can use.  Also, I'm not sure why it is named that in the Hive operator... the code was clearly copied from Raptor.

Reply:
@dain is correct, this is a synthetic column (not queryable) used for row-level deletes (which actually aren't implemented for Hive but due to the way deletes work in the engine we need to return a column handle for it anyway). And it was misnamed due to being copied from Raptor.

Reply:
I will remove it then